docs/tour/hello-world.mdx

```diff
@@ -77,8 +77,9 @@ brew install go
 ```
 
 ```shell
-brew tap tinygo-org/tools
-brew install tinygo
+brew tap-new tinygo/tap
+brew extract --version 0.33.0 tinygo-org/tools/tinygo tinygo/tap
+brew install tinygo/tap/tinygo@0.33.0
```

Comment:
```suggestion
brew install tinygo/tools/tinygo@0.33.0
```

Reply:
```
Warning: No available formula or cask with the name "tinygo-org/tools/tinygo@0.33.0". Did you mean tinygo-org/tools/tinygo?
```

Sadly, this is why we need the new tap

Reply:
oh right, my brain didn't catch that. Yep, ignore my suggestion :)